src/main/java/com/aws/greengrass/clientdevices/auth/metrics/ClientDeviceAuthMetrics.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.metrics;
+
+
+import com.aws.greengrass.telemetry.impl.Metric;
+import com.aws.greengrass.telemetry.models.TelemetryAggregation;
+import com.aws.greengrass.telemetry.models.TelemetryUnit;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class ClientDeviceAuthMetrics {
+
+    private static AtomicLong certSubscribeSuccess = new AtomicLong();
```

Comment:
```suggestion
    private final AtomicLong certSubscribeSuccess = new AtomicLong();
```
Would avoid static here unless we have a specific reason for it.  Generally, static state can cause flaky unit tests if we're not careful